Dockerfile

```diff
@@ -22,7 +22,7 @@ RUN mkdir -p /srv/vets-api/{clamav/database,pki/tls,secure,src} && \
 # XXX: get rid of the CA trust manipulation when we have a better model for it
 COPY config/ca-trust/* /usr/local/share/ca-certificates/
 # rename .pem files to .crt because update-ca-certificates ignores files that are not .crt
```

Comment:
does this comment make sense with the change?

Reply:
yep! I just had the logic flipped in the `if`